05-tcache/tcache_house_of_spirit.c

```diff
@@ -25,22 +25,22 @@ typedef struct malloc_chunk* mchunkptr;
 int main(int argc, const char* argv[]) {
     size_t fake_chunk_and_more[64];
     
-    printf("This example showcases how the House of Spirite became more powerful " \
+                        printf("This example showcases how the House of Spirite became more powerful " \
             " after the tcache patch\n");
 
-    printf("Filling space at and after the fake chunk with invalid data\n");
-    memset(fake_chunk_and_more, 'A', sizeof(fake_chunk_and_more));
+        printf("Filling space at and after the fake chunk with invalid data\n");
+            memset(fake_chunk_and_more, 'A', sizeof(fake_chunk_and_more));
 
     printf("Building fake chunk on the stack at %p\n", (void *)fake_chunk_and_more);
-    mchunkptr fake_chunk = (mchunkptr)(void *)fake_chunk_and_more;
-    fake_chunk->size = 0x110;
+                    mchunkptr fake_chunk = (mchunkptr)(void *)fake_chunk_and_more;
+                                                                                    fake_chunk->size = 0x110;
```

Comment:
I believe this line is a bit excessive, what do you think?
![image](https://user-images.githubusercontent.com/9214393/28056418-295ed00e-661d-11e7-94cc-0fb390a60396.png)